client/src/GamemasterInGame/Lobby.tsx

```diff
@@ -116,6 +132,29 @@ export function Lobby() {
                 </Tabs.Content>
               </motion.div>
             )}
+            {selectedTab === "menu" && (
+              <motion.div
+                className="absolute h-full w-full"
+                key="menu"
+                initial={{ x: "100%", opacity: 0.3 }}
+                animate={{ x: 0, opacity: 1 }}
+                exit={{ x: "100%", opacity: 0.3 }}
```

Comment:
The trouble with tab animations is now that we have three tabs, the middle tab isn't going to enter/exit the correct direction.  We'll either have to have two selectedTab states (current and prev) to track for the middle tab or just remove animations entirely for now.